tox.ini

```
@@ -1,8 +1,9 @@
 [tox]
 envlist = py3, pypy3, doc, pep8
+isolated_build = True
 
 [testenv]
-deps=-r test-requirements.txt
+deps=dev
```

```suggestion
extras=dev
```

@hugovk 's suggestion was not applied correctly, which seems to be the cause of the build failure